modules/middleware/src/flattenAggregations.js

```diff
@@ -50,4 +62,10 @@ function flattenAggregations({ aggregations, includeMissing = true }) {
   }, {});
 }
 
+const termFilterFields = values => {
+  const regex = RegExp('([a-zA-Z0-9._]*).term_filter$');
```

Comment:
I would extract this RegExp to avoid creating a new object each time the function is invoked.